jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -130,6 +130,7 @@ public static Containerizer to(TarImage tarImage) {
   private boolean allowInsecureRegistries = false;
   private boolean offline = false;
   private String toolName = DEFAULT_TOOL_NAME;
+  private boolean alwaysCacheBaseImage = true;
```

Comment:
Like I said in another comment, this should probably default to `false` not `true`.